cypress.config.js

```diff
@@ -2,8 +2,10 @@ const { defineConfig } = require('cypress');
 
 module.exports = defineConfig({
   e2e: {
+    baseUrl: 'https://demoqa.com/webtables',
     setupNodeEvents(on, config) {
-      // implement node event listeners here
+      on('task', {
+      });
```

Comment:
The 'task' event listener is empty. Please implement the required functionality inside the listener.